api/Dockerfile

```diff
@@ -4,57 +4,57 @@
 
 
 # https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
-ARG PHP_VERSION=8.0
-ARG CADDY_VERSION=2.4.3
+ARG PHP_VERSION=8.1
+ARG CADDY_VERSION=2
```

Comment:
Das nicht mehr genau pinnen ist von api platform übernommen?

Reply:
Ah ich hatte in der Vergangenheit schon Probleme beim Caddy Container builden wenn das nicht genau gepinnt ist. Müssen wir einfach im Kopf behalten, und wenn der Build mal wieder nicht funktioniert die Version wieder genau pinnen.

Reply:
Ja, hab ich so einfach von API platform übernommen. Hab's jetzt auf 2.4.6 gepinnt. Dürfen wir einfach nicht vergessen, von Zeit zu Zeit zu aktualisieren. Renovate kann das ja nicht, oder?

Reply:
Wir können es Renovate beibringen, ähnlich wie in #2380 renovate.json so geändert wurde dass die PHP-Version in dieser Datei geupdatet werden kann.

Reply:
Done with https://github.com/ecamp/ecamp3/pull/2370/commits/924d03f48016806aaa9fde34e8dc02d08a6c6c59